conda-forge.yml

```
@@ -1,6 +1,12 @@
-github:
-  branch_name: main
-  tooling_branch_name: main
+azure:
+  free_disk_space: true
+#build_platform:  # force full emulation to resolve segfault from some libs
+#  linux_aarch64: linux_64
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+github:
+  branch_name: main
+  tooling_branch_name: main
+provider:
+  linux_aarch64: default
```

Also good for us to be explicit and include `linux_64`

```suggestion
provider:
  linux_64: default
  linux_aarch64: default
```

---

Will do but in a new PR because holoscan team is OK for us to support `sbsa` starting with the latest GA version which is now `3.1.0.3`. I will refer to this PR from that, and close this PR later.